vm/systemSmartContracts/delegation.go

```diff
@@ -717,6 +1149,11 @@ func (d *delegation) createAndSaveNextFund(address []byte, value *big.Int, fundT
 }
 
 func (d *delegation) saveFund(key []byte, dFund *Fund) error {
```

Comment:
delegationFund ?

Reply:
it is general fund- would leave as it is.

Reply:
ok